### PR TITLE
[zh] sync nodes addons sidecar-containers

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -1076,6 +1078,8 @@ Learn more about the following:
 * 架构设计文档中有关
   [Node](https://git.k8s.io/design-proposals-archive/architecture/architecture.md#the-kubernetes-node)
   的章节。
+* [集群自动伸缩](https://git.k8s.io/design-proposals-archive/architecture/architecture.md#the-kubernetes-node)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:1081]:
```suggestion
* [集群自动扩缩](https://git.k8s.io/design-proposals-archive/architecture/architecture.md#the-kubernetes-node)
```

File: content/zh-cn/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -209,6 +209,15 @@ Add-on 扩展了 Kubernetes 的功能。
   并将系统问题报告为[事件](/zh-cn/docs/reference/kubernetes-api/cluster-resources/event-v1/)
   或[节点状况](/zh-cn/docs/concepts/architecture/nodes/#condition)。
 
+<!--
+## Instrumentation
+
+* [kube-state-metrics](/docs/concepts/cluster-administration/kube-state-metrics)
+-->
+## 监控  {#instrumentation}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/addons.md:217]:
```suggestion
## 插桩  {#instrumentation}
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**